src/colmap/sfm/incremental_mapper.cc

```diff
@@ -798,7 +796,6 @@ void IncrementalMapper::IterativeGlobalRefinement(
       break;
     }
   }
-  FilterImages(options);
```

Comment:
We now lost the ability to remove degenerate images here?

Reply:
I moved it to the parent scope so it doesn't get called when triangulating only
https://github.com/colmap/colmap/blob/1ff3857f15c9df5143d90b63bf4f64945ced9375/src/colmap/controllers/incremental_mapper.cc#L47